redhat/rdma-core.spec

```diff
@@ -354,10 +354,11 @@ rm -rf %{buildroot}/%{_initrddir}/
 rm -rf %{buildroot}/%{_sbindir}/srp_daemon.sh
 
 %post -n rdma-core
-# we ship udev rules, so trigger an update.
```

Comment:
Oh there it is, please squash this into the earlier patch

Reply:
OK.